R/subset_wuse.R

```diff
@@ -17,13 +17,13 @@
 #' areas <- "10" # NA uses all areas
 #' area.column <- "STATECODE"
 #' 
-#' w.wuse.sub <- subset_wuse(w.use, data.elements,area.column,areas)
+#' w.use.sub <- subset_wuse(w.use, data.elements,area.column,areas)
 subset_wuse <- function(w.use, data.elements, area.column, areas=NA) {
   # subset w.use on basis of areas and select the data elements of interest
   if (all(is.na(areas))){
-    w.use.sub <- w.use[,c("YEAR",area.column,data.elements)]
+    w.use.sub <- w.use[,c("YEAR","STATECOUNTYCODE",area.column,data.elements)]
```

Comment:
Agreed: we'll need a way to identify what kind of spatial dataset the user is dealing with. For the purposes of v 0.1.0 I figured we'd set up the choropleth function to strictly work with FIPS for now and add that flexibility on the way to 1.0.0. Does that sound reasonable?

It's seems the choropleth function could append ```STATECOUNTYCODE``` to ```data.elements``` to retrieve the codes (that way we don't mess with the ```subset_wuse``` function), but I could be missing something ... if not we can write a separate function. When we work in checks for spatial data type, we can ensure that the element should/will be present in the WU data.